bluemira/codes/process/_inputs.py

```diff
@@ -0,0 +1,303 @@
+    gapomin: float = 0.2
+    hfact: float = 1.1
+    hldivlim: float = 10.0
+    i_single_null: int = 1
+    i_tf_sc_mat: int = 5
+    i_tf_turns_integer: int = 1
+    iavail: int = 0
+    ibss: int = 4
+    iculbl: int = 1
+    icurr: int = 4
+    idensl: int = 7
+    iefrf: int = 10
+    ieped: int = 1
+    ifalphap: int = 1
+    ifispact: int = 0
+    ifueltyp: int = 1
+    iinvqd: int = 1
+    impvar: int = 13
+    inuclear: int = 1
+    iohcl: int = 1
+    ioptimz: int = 1
+    ipedestal: int = 1
+    ipfloc: List[int] = field(default_factory=lambda: [2, 2, 3, 3])
+    ipowerflow: int = 0
+    iprimshld: int = 1
+    iprofile: int = 1
+    isc: int = 34
+    ishape: int = 0
+    isumatoh: int = 5
+    isumatpf: int = 3
+    kappa: float = 1.848
+    ksic: float = 1.4
+    lpulse: int = 1
+    lsa: int = 2
+    minmax: int = 1
+    n_layer: int = 10
+    n_pancake: int = 20
+    n_tf: int = 16
+    ncls: List[int] = field(default_factory=lambda: [1, 1, 2, 2])
+    neped: float = 6.78e19
+    nesep: float = 2e19
+    ngrp: int = 4
+    oacdcp: float = 8673900.0
+    oh_steel_frac: float = 0.57875
+    ohcth: float = 0.55242
+    ohhghf: float = 0.9
+    output_costs: int = 0
+    pheat: float = 50.0
+    pinjalw: float = 51.0
+    plasma_res_factor: float = 0.66
+    pnetelin: float = 500.0
+    primary_pumping: int = 3
+    prn1: float = 0.4
+    psepbqarmax: float = 9.2
+    pulsetimings: float = 0.0
+    q0: float = 1.0
+    q: float = 3.5
+    qnuc: float = 12920.0
+    ralpne: float = 0.06894
+    rhopedn: float = 0.94
+    rhopedt: float = 0.94
+    ripmax: float = 0.6
+    rjconpf: List[float] = field(
+        default_factory=lambda: [1.1e7, 1.1e7, 6e6, 6e6, 8e6, 8e6, 8e6, 8e6]
+    )
+    rmajor: float = 8.8901
+    rpf2: float = -1.825
+    scrapli: float = 0.225
+    scraplo: float = 0.225
+    secondary_cycle: int = 2
+    shldith: float = 1e-06
+    shldoth: float = 1e-06
+    shldtth: float = 1e-06
+    sig_tf_case_max: float = 580000000.0
+    sig_tf_wp_max: float = 580000000.0
+    ssync: float = 0.6
+    tbeta: float = 2.0
+    tbrnmn: float = 7200.0
+    tburn: float = 10000.0
+    tdmptf: float = 25.829
+    tdwell: float = 0.0
+    te: float = 12.33
+    teped: float = 5.5
+    tesep: float = 0.1
+    tfcth: float = 1.208
+    tftmp: float = 4.75
+    tftsgap: float = 0.05
+    thicndut: float = 0.002
+    thkcas: float = 0.52465
+    thwcndut: float = 0.008
+    tinstf: float = 0.008
+    tlife: float = 40.0
+    tmargmin: float = 1.5
+    tramp: float = 500.0
+    triang: float = 0.5
+    ucblvd: float = 280.0
+    ucdiv: float = 500000.0
+    ucme: float = 300000000.0
+    vdalw: float = 10.0
+    vfshld: float = 0.6
+    vftf: float = 0.3
+    vgap2: float = 0.05
+    vvblgap: float = 0.02
+    walalw: float = 8.0
+    zeffdiv: float = 3.5
+    zref: List[float] = field(
+        default_factory=lambda: [3.6, 1.2, 1.0, 2.8, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
+    )
+
+    def __iter__(self) -> Generator[field, None, None]:
```

Comment:
`field` is a function, I think. We may want `Field`, but aren't they `INVariable`s?

Reply:
When this is first called they are not INVariables (the start of the post_init) and then I use the field name to create the INVariable.